### PR TITLE
Base Mobile Project

### PR DESCRIPTION
**Pull request by jessepierce451**
Still lots to do
- Create a README for navigating all of this
- Implement Tailwind with twrnc
- Setup Structure
- IF we plan on a web app version, a good pattern is to decouple logic from this app into a consumable js module for both web and mobile, a bonsai-balance.core if you will
- Setup Typescript

### CODE REVIEW

File: src/BonsaiBalance.Mobile/BaseProject/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  arrowParens: 'avoid',
+  bracketSameLine: true,
+  bracketSpacing: false,
+  singleQuote: true,
+  trailingComma: 'all',
```
Comment (FullofQuarks) [inline src/BonsaiBalance.Mobile/BaseProject/.prettierrc.js:6]: Not sure I'm a fan of trailing commas

File: src/BonsaiBalance.Mobile/BaseProject/.ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.7.5
```
Comment (FullofQuarks) [inline src/BonsaiBalance.Mobile/BaseProject/.ruby-version:1]: Ruby?

File: src/BonsaiBalance.Mobile/BaseProject/.watchmanconfig [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (FullofQuarks) [inline src/BonsaiBalance.Mobile/BaseProject/.watchmanconfig:1]: ?

**Review by FullofQuarks — APPROVED** [review]

File: src/BonsaiBalance.Mobile/BaseProject/.watchmanconfig (hunk 3 above)
Comment (jessepierce451) [reply src/BonsaiBalance.Mobile/BaseProject/.watchmanconfig:1]: that should probably be in the gitignore

File: src/BonsaiBalance.Mobile/BaseProject/.ruby-version (hunk 2 above)
Comment (jessepierce451) [reply src/BonsaiBalance.Mobile/BaseProject/.ruby-version:1]: Ruby is for cocoapods, which is for iOS stuff

**Pull request merged**